src/component/common/Table/SortableTableHeader/CellSortable/CellSortable.tsx

```diff
@@ -42,28 +58,68 @@ export const CellSortable: FC<ICellSortableProps> = ({
         );
     };
 
+    const justifyContent = useMemo(() => {
+        switch (align) {
+            case 'left':
+                return 'flex-start';
+            case 'center':
+                return 'center';
+            case 'right':
+                return 'flex-end';
+            default:
+                return undefined;
+        }
+    }, [align]);
+
+    useEffect(() => {
+        const updateTitle = () => {
+            const newTitle =
+                ariaTitle &&
+                ref.current &&
+                ref?.current?.offsetWidth < ref?.current?.scrollWidth
+                    ? `${children}`
+                    : '';
+
+            if (newTitle !== title) {
+                setTitle(newTitle);
+            }
+        };
+
+        updateTitle();
+    }, [ref.current, setTitle, ariaTitle]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I don't think changing the refs value here will trigger a re-render. See: 

https://epicreact.dev/why-you-shouldnt-put-refs-in-a-dependency-array/